### ISSUE TITLE
Add superscripts for Greek letters

### ISSUE THREAD

**Issue by DenverCoder1** [DenverCoder1/LaTeX-Gboard-Dictionary#55]
Please read the formatting guidelines explained in #5 to contribute!

When submitting, you may choose to leave some symbols for others to contribute.

Before submitting a PR check:

* There are no active pull requests for the symbol you are submitting
* You have proper tabs between characters as explained in #5 (you can copy paste tab symbol from other lines)

### Symbols to add

These are the only Greek letters supporting superscript as far as I know:

```
^Phi	ᶲ	
^alpha	ᵅ	
^beta	ᵝ	
^chi	ᵡ	
^delta	ᵟ	
^epsilon	ᵋ	
^gamma	ᵞ	
^iota	ᶥ	
^phi	ᵠ	
^theta	ᶿ	
```

**Comment by yash2002109** [comment]
Hi @DenverCoder1, I'd like to work on this issue. Could you please assign it to me?

**Comment by DenverCoder1** [comment]
@yash2002109 You've made a couple contributions today, maybe it would be best to allow others to make their first contributions as well?

**Comment by yash2002109** [comment]
Yes, I agree with you. We can let someone else handle the issue!

**Comment by eshaanagarwal** [comment]
Hi @DenverCoder1, I am interested in working on this issue. Could you please assign it to me?

**Comment by DenverCoder1** [comment]
@eshaanagarwal That would be great :+1: Feel free to open a PR, you may decide to add one or more of them. Preferably leaving some for others to contribute :)
